src/app/scripts/cac/control/cac-control-sidebar-explore.js

```diff
@@ -60,15 +75,45 @@ CAC.Control.SidebarExplore = (function ($, MapTemplates, Typeahead, UserPreferen
 
     return SidebarExploreControl;
 
+    function changeMode() {
+        bikeOptions.changeMode(options.selectors);
+    }
+
     /**
      * Set user preferences before fetching isochrone.
      */
     function clickedExplore() {
         var exploreMinutes = $(options.selectors.exploreTime).val();
-        var mode = $(options.selectors.exploreMode).val();
+        var mode = $(options.selectors.modeSelector).val();
+
+        var picker = $(options.selectors.datepicker).data('DateTimePicker');
+        var date = picker.date();
+        if (!date) {
+            // use current date/time if none set
+            date = moment();
+        }
+
+        var otpOptions = { mode: mode };
+
+        if (mode.indexOf('BICYCLE') > -1) {
+            var bikeTriangleOpt = $('option:selected', options.selectors.bikeTriangleDiv);
```

Comment:
Seems like the BikeControl is still leaky here, since the selectors are used both in the bike control and outside it. If you expose methods on the bikecontrol that provide the three values you're pulling, then the change mode method doesn't need the selectors argument and you can just set the selectors as configurable defaults in that control.